scripts/mkcloud

```diff
@@ -580,9 +580,9 @@ function setuplonelynodes()
         local lonely_node
         lonely_node=$cloud-node$i
         ${mkcloud_lib_dir}/libvirt/compute-config $cloud $i $mac 0\
-            "$cephvolume" "$drbdvolume" $compute_node_memory\
+            "$cephvolumenumber" "$drbdvolume" $compute_node_memory\
             $controller_node_memory $libvirt_type $vcpus $emulator $vdisk_dir\
-            1 1 > /tmp/$cloud-node$i.xml
+            1 1 > /tmp/$cloud-node$i.xml || exit 5
```

Comment:
@jdsn @MaximilianMeister  fixed in https://github.com/toabctl/automation/commit/9aac40f0b15aafb5519e78172084f909f175cb17